content/news/playtest-20150719.md

```diff
@@ -0,0 +1,46 @@
+We are pleased to announce the second playtest on the road to our next stable release.  With this playtest, we have entered the feature freeze phase, so it already is pretty close to what the release will eventually look like. There are probably a number of bugs still lurking in the code, so it needs testing, testing and some more testing!
+
+Compared to the previous playtest, there have been a number of new features, fixes and other changes all over the place. Music playback has received quite a number of changes. Maps can now define a soundtrack, which defaults to a random track, and also separate soundtracks for victory and defeat which are played on the score screen.
+
+In the multi-player lobby, selecting a map which contains custom rules will issue a warning message in the chat window to alert the players to the fact.
+
+We have also added some features that people watching replays, spectators and game casters will benefit from. Range circles of all kinds are now visible not only to the owner of the unit or building, but to allies and spectators as well. You are now also able to hide the in-game UI elements with a single hotkey (Ctrl+Shift+H by default).
+
+In-game, a noteworthy change is that paratroopers can now be selected and given orders while they are still in the air. Spies have also received a major overhaul. They are now able to detect other enemy spies as well as cloaked units, and infiltrating enemy production buildings enables you to build veteran units. These changes make spies more useful overall on the battlefield.
```

Comment:
All these paragraphs are a big wall of text, which we know turns off casual readers.  Can we defer some of these to the bullet points below?